dist/templates/rbac-ovnkube-master.yaml.j2

```diff
@@ -158,3 +158,35 @@ rules:
     - apiGroups: [""]
       resources: ["configmaps"]
       verbs: ["create", "patch", "update"]
+
+---
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
@trozet just as a reminder, this will require a separate PR downstream(CNO).

Reply:
i also need to get the e2es backward compatible downstream before we merge this